client/router_create.go

```diff
@@ -446,6 +446,11 @@ func ClusterRoles() []*rbacv1.ClusterRole {
 				Resources: []string{"nodes"},
 				Verbs:     []string{"get", "list", "watch"},
 			},
+			{
+				Verbs:     []string{"watch", "create", "patch"},
```

Comment:
Out of curiosity, why do we need a cluster scoped permission for events?

Reply:
I will remove this. This pr only send events from the service-controller so it won't be necessary to have those permissions. Thanks!